op-program/client/driver/program.go

```diff
@@ -62,10 +62,6 @@ func (d *ProgramDeriver) OnEvent(ev event.Event) bool {
 			d.logger.Info("Derivation complete: reached L2 block", "head", x.SafeL2Head)
 			d.closing = true
 		}
-	case derive.DeriverIdleEvent:
-		// Not enough data to reach target
-		d.closing = true
-		d.logger.Info("Derivation complete: no further data to process")
```

Comment:
We could also just remove the actual state change, but retain the info log. It may be interesting to know that derivation completed.
```go
	case derive.DeriverIdleEvent:
		d.logger.Info("Derivation complete: no further data to process")
```

Reply:
I'll add it back in the holocene action test PR

Reply:
I think the log message there is misleading now. We haven't completed derivation yet. The log message is now in driver.go which is where we actually detect the derivation is complete. Based on comments elsewhere I think you wound up noticing that but commenting here for the record. :)